vendor.conf

```diff
@@ -1,7 +1,7 @@
 github.com/containers/image
 
 github.com/sirupsen/logrus v1.0.0
-github.com/containers/storage master
+github.com/containers/storage 02db7cbb0bb082681f9855131fec8c8e1a3f4cb6
```

Comment:
This will freeze the version of c/storage in our CI. Do you want that, and for c/image CI to only update by an explicit decision? (I don’t have a strong opinion either way.)

Reply:
I don't have a strong opinion, either.  Dropping this.